server/db/migrate/20160527090120_rename_title_key.rb

```diff
@@ -0,0 +1,16 @@
+class RenameTitleKey < ActiveRecord::Migration
+  def change
+    # Copy values from :en_jp to new key :ja_en, update canonical_title values
+    execute <<-SQL
+      UPDATE franchises
+      SET titles = titles || hstore('ja_en', titles -> 'en_jp'), canonical_title = 'ja_en'
+    SQL
+    # Delete the old :en_jp key
+    execute <<-SQL
+      UPDATE franchises
+      SET titles = delete(titles, 'en_jp')
+    SQL
+    # Update canonical_title default
+    change_column_default :franchises, :canonical_title, 'ja_en'
```

Comment:
I did an update on canonical_title in the first UPDATE statement. Seemed like the field was added in the previous migration so I assumed all fields would still be the default (`en_jp`).